test/e2e/software_upgrade_e2e_v3_test.go

```diff
@@ -183,7 +183,9 @@ func (s *SoftwareUpgradeV3TestSuite) TestUpgradeV3() {
 
 	maxFP, ok := btcparamsMap["max_finality_providers"]
 	s.Require().True(ok, "max_finality_providers param should exist")
-	s.Require().Equal(float64(1), maxFP, "max_finality_providers should be 1")
+	s.Require().Equal(float64(5), maxFP, "max_finality_providers should be 5")
+
+	s.Require().Equal(float64(288), btcparamsMap["btc_activation_height"], "BtcActivationHeight should be incremented correctly")
```

Comment:
this is currently set to `260000` in testent upgrade, so test fill fail after backport